torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2375,7 +2370,7 @@ def create_unspecified_symbol(
 
         # We don't want to specialize zero one val for unspecified symbol
         # so that we can always get a new symbol despite val.
-        return self.create_symbol(val, source, dynamic_dim, constraint_dim, positive=None, do_not_specialize_zero_one=True)
+        return self.create_symbol(val, source, dynamic_dim, constraint_dim, positive=None, do_not_specialize_zero_one=True, symbolic_context=None)
 
     @record_shapeenv_event()
     def create_symbol(
```

Comment:
so now we're caching at the `create_symbol()` level instead of at the `create_symintnode()` level, correct?

does `create_symintnode()` still need to take in a `symbolic_context`? doesn't look like it's used

Reply:
rm